### PR TITLE
added mongodb PHP7 beta driver.

### PR DESCRIPTION
**Pull request by morozovsk**
Work only insert, update, list database, list index, list tables. select, where, limit, order, delete, edit, edit from select.
does not work structure modify, index, modify, custom queries, export, import, groups, collections drop, add.

### CODE REVIEW

**Comment by morozovsk** [comment]
it has nothing in common: https://gist.github.com/morozovsk/14374da65d88f32f94ad061f4a7b942c/revisions

File: adminer/drivers/mongo.inc.php [hunk 1]
```diff
@@ -354,4 +353,614 @@ function support($feature) {
...
+						}
+
+					}
+				}
+			}
+
+			return $data;
+		}
+
+		$jush = "mongodb";
```
Comment (vrana) [inline adminer/drivers/mongo.inc.php]: Use `mongo`.

**Comment by morozovsk** [comment]
>Most of the global functions (connect(), error(), …, support()) and variables ($functions, …, $edit_functions) seem to be copy/pasted.

I see you prefer 100 "if/else" than one "if/else".
>Use mongo.

-> use php5
>mongo is legacy now: 

http://php.net/manual/en/book.mongo.php
>This extension is deprecated. Instead, **the MongoDB extension should be used**.

http://php.net/manual/en/mongo.installation.php

**Comment by vrana** [comment]
Put all common functions in a common block. Put extension-specific functions in the `if` block.

JUSH is a syntax highlighter, `$jush` holds the name of the language. It has nothing in common with PHP version or extensions.

**Comment by morozovsk** [comment]
done

**Comment by vrana** [comment]
Merged, thanks.

**Comment by morozovsk** [comment]
Thank you @pivolan for your help!

**Pull request closed without merging**